person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age?
+
+    false
+  end
```

Comment:
- [ ] Kindly make sure that this method returns `true` if the `of_age?` returns `true`, _**or if the person has parent permission**_, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=or%20if%20they%20have%20permission%20from%20parents.) requirement.